protocols/v2/codec-sv2/src/decoder.rs

```diff
@@ -157,17 +233,24 @@ impl<'a, T: Serialize + GetSize + Deserialize<'a>, B: IsBuffer + AeadBuffer> Wit
         frame.into()
     }
 
+    /// Provides a writable buffer for incoming data.
```

Comment:
Maybe I would add that the buffer have the right number of bytes needed by the codec to go to the next step. I would also add an example of how to use the returned buffer.